src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -43,6 +43,54 @@ interface AuthorizeService {
     completeSignOut(url: string): Promise<AuthenticationResult>;
 }
 
+// These are the values for the .NET logger LogLevel. 
+// We only use debug and trace
+export enum LogLevel {
+    Trace = 0,
+    Debug = 1
+}
+
+interface JavaScriptLoggingOptions {
+    debugEnabled: boolean;
+    traceEnabled: boolean;
+}
+
+export class Logger {
```

Comment:
Nit: I'm probably missing something, but why are these types `export`ed? Are they going to be consumed elsewhere?

Reply:
They are not consumed outside the lib, they could technically not be exported, but it does not matter.